ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -207,7 +225,42 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
                 <SlidingPanel isShown={!!sidePanel} onClose={() => setSidePanel(null)}>
                     {sidePanel === 'share' && <WidgetGallery namespace={namespace} label={'workflows.argoproj.io/cron-workflow=' + name} />}
                 </SlidingPanel>
+                <>
+                    <ErrorNotice error={error} />
+                    {!workflows ? (
+                        <ZeroState title='No completed cron-workflows'>
+                            <p> You can submit or resume cron workflow to run a cron-workflow. </p>
```

Comment:
```suggestion
                        <ZeroState title='No completed cron workflows'>
                            <p>You can create new cron workflows here or using the CLI.</p>
```

matching the wording on [the `CronWorkflowList` `ZeroState`](https://github.com/argoproj/argo-workflows/blob/9b7b78058c632315f86f59d081d6f6ef6cddbc9b/ui/src/app/cron-workflows/components/cron-workflow-list/cron-workflow-list.tsx#L121) would be preferred for consistency.